OpenConsole.sln

```diff
@@ -232,6 +232,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Shared", "Shared", "{89CDCC
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Types.Unit.Tests", "src\types\ut_types\Types.Unit.Tests.vcxproj", "{34DE34D3-1CD6-4EE3-8BD9-A26B5B27EC73}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "PublicTerminalCore", "src\cascadia\PublicTerminalCore\PublicTerminalCore.vcxproj", "{84848BFA-931D-42CE-9ADF-01EE54DE7890}"

```

Comment:
I'm not sure I understand the naming here as "Public" TerminalCore. TerminalCore is already interfaces, isn't it? And this doesn't seem to re-use anything from it...

Reply:
I had trouble thinking of a good name for it, this is a C-API exposed set of terminal APIs

Reply:
OK. I can't think of a better one either for now. There's always rename in the future.